spotless fails running on windows

Since #981 it is impossible to build spotbugs on Windows, due spotless errors like:
```
> Task :eclipsePlugin:spotlessJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':eclipsePlugin:spotlessJava'.
> The following files had format violations:
      eclipsePlugin\src\de\tobject\findbugs\actions\ClearMarkersAction.java
          @@ -1,80 +1,80 @@
          -/*\n
          - * FindBugs Eclipse Plug-in.\n
          - * Copyright (C) 2003 - 2004, Peter Friese\n
          - *\n
          - * This library is free software; you can redistribute it and/or\n
          - * modify it under the terms of the GNU Lesser General Public\n
          - * License as published by the Free Software Foundation; either\n
          - * version 2.1 of the License, or (at your option) any later version.\n
          - *\n
          - * This library is distributed in the hope that it will be useful,\n
          - * but WITHOUT ANY WARRANTY; without even the implied warranty of\n
          - * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU\n
          - * Lesser General Public License for more details.\n
          - *\n
          - * You should have received a copy of the GNU Lesser General Public\n
          - * License along with this library; if not, write to the Free Software\n
          - * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA\n
          - */\n
          -\n
          -package de.tobject.findbugs.actions;\n
          -\n
          -import java.util.List;\n
          -\n
          -import org.eclipse.core.resources.IResource;\n
          -import org.eclipse.core.runtime.CoreException;\n
          -import org.eclipse.core.runtime.IProgressMonitor;\n
          -import org.eclipse.core.runtime.jobs.IJobChangeEvent;\n
          -import org.eclipse.core.runtime.jobs.JobChangeAdapter;\n
          -import org.eclipse.ui.IWorkbenchPart;\n
          -\n
          -import de.tobject.findbugs.FindBugsJob;\n
          -import de.tobject.findbugs.builder.WorkItem;\n
          -\n
          -/**\n
          - * Remove all bug markers for the given selection.\n
          - *\n
          - * @author Peter Friese\n
          - * @author Phil Crosby\n
          - * @author Andrei Loskutov\n
          - * @version 2.0\n
          - * @since 25.09.2003\n
          - */\n
          -public class ClearMarkersAction extends FindBugsAction {\n
          -\n
          -    /**\n
          -     * Clear the FindBugs markers on each project in the given selection,\n
          -     * displaying a progress monitor.\n
          -     */\n
      ... (112 more lines that didn't fit)
  Violations also present in 130 other files.
  Run 'gradlew spotlessApply' to fix these violations.
```

Looks like https://github.com/diffplug/spotless/tree/master/plugin-gradle wants everyone to have either .gitattributes with crlf settings or explicit `lineEndings 'UNIX'` property set.

Since we assume we want UNIX / LF by default anyway, I will set it in the spotless.gradle by default to be able to build on Windows.

--- a/gradle/spotless.gradle
+++ b/gradle/spotless.gradle
@@ -5,5 +5,6 @@ spotless {
     removeUnusedImports()
     eclipse().configFile "${rootProject.projectDir}/gradle/eclipse-formatter.xml"
     endWithNewline()
+    lineEndings 'UNIX'
   }
 }

